tests/JSONSchemaGenerator.test.js

```diff
@@ -75,7 +80,20 @@ tap.test('generate JSON Schemas', t => {
         t.plan(2)
         const method = operationToMethod[operation]
         const schema = generator[method]()
-        t.strictSame(schema, expectedSchemas[operation][collection])
+        if (collection.endsWith('New')) {
```

Comment:
Because to fix the date problem in the swagger I had to generate a different schema in the new configuration. This means that new configuration will never have the same config as the old one in Date cases. Btw I still wanted to mantain this tests in order to check the correct generation of other fields

Reply:
OK, but this is not clear from the code though.
Can we just create a new expected schema specific for the new collections in the expected schema folder? In this way it would be clearer to understand what is tested.

Reply:
I could, but in this way we'll lose the comparison between old and new configurations. On the other hand, by doing this, we lose just the comparison between old and new date fields

Reply:
What do you think about it?